openml/testing.py

```diff
@@ -65,6 +65,10 @@ def setUp(self):
                 with open(openml.config.config_file, 'w') as fh:
                     fh.write('apikey = %s' % openml.config.apikey)
 
+        # Increase the number of retries to avoid spurios server failures
+        self.connection_n_retries = openml.config.connection_n_retries
+        openml.config.connection_n_retries = 10
```

Comment:
I'm a bit confused about the number of retries being stored in the _default dict, the openml.config file and self.connection_n_retries. So, by default it is 2, then when that doesn't work, it is increased to 10 and stored in the openml.config, and then it stops (it doesn't get higher than 10)?

Reply:
Nope, it's increased to 10 for testing in every case. It is stored in the class to restore the original value afterwards.

Reply:
I agree that it is unfortunate that the value of 2 is twice in the original file, I just made a note to change it later.